src/cluster.c

```diff
@@ -5709,6 +5709,10 @@ void readonlyCommand(client *c) {
 
 /* The READWRITE command just clears the READONLY command state. */
 void readwriteCommand(client *c) {
+    if (server.cluster_enabled == 0) {
```

Comment:
In non cluster, you can't enter readonly mode, so this check is redundancy.

Reply:
Yes, i know. Here is my thought, `readwrite` quite like `not readonly`. Althought `readwrite` can be executed in non-cluster mode, it does not do anything actually.

This check can added or not, it will not change anthing about the execute. But in my opinion, it can treat like `not readonly`, and maybe it should act like `readonly` when it execute in non-cluster mode. So when the user type `readwrite` in non-cluster mode, it can get the same error.

Reply:
I think it would make sense to make this symmetric.

Reply:
Also make sense to me, agree with the `symmetric`